outlines/accounts.md

```diff
@@ -24,15 +24,17 @@
     6. Meetup
     7. Meteor Developer Accounts
     8. Building your own OAuth login handler
-5. The useraccounts package for a production-grade login UI
-    1. Pick the right package based on your CSS framework
-    2. Get help from splendido to flesh out this section - just the basics, then link to the docs
-    3. Figure out what to do about useraccounts and adding fields to profile
-6. Adding custom data about users
+5. Adding custom data about users
     1. Adding new top-level fields onto the users collection
     2. How and why to disable profile
     3. Why top-level fields are better than nested objects in DDP (link to collections/schema article)
     4. Security concerns - don't accidentally publish secret data to the client
+6. The useraccounts package for a production-grade login UI
+    1. Rationale (logic & routing vs. styling)
```

Comment:
a simple introduction with a very few lines explaining the organization of the packages (_core_ provides the logic which is common for every other package providing styles for the different UI frameworks and the routing packages; one of the goals is customization; etc...)

Feel free to suggest a different wording (I'm not native English..)

It might also be that 6.i is not needed as a standalone section and the same text could be put under 6.